contracts/libraries/SafeToL2Setup.sol

```diff
@@ -71,7 +71,7 @@ contract SafeToL2Setup is SafeStorage {
     /**
      * @notice Returns the current chain ID.
      */
-    function chainId() private view returns (uint256 result) {
```

Comment:
using `view` is correct here, because according to the definition:
> The difference between view and pure is that view declares that a function can read from the blockchain but cannot modify the state of the contract, while pure declares that a function cannot read from the blockchain or modify the state of the contract.

Taken from https://medium.com/coinmonks/rareskills-solidity-interview-question-25-answered-what-is-the-difference-between-view-and-pure-7ccc6bce1704

Here, we're reading the chain ID from the blockchain's state.

Reply:
(I know about the warning, but in this case, it's a false positive; you can add an ignore warning statement instead to suppress it.)

Reply:
fair enough, reverted the the state mutability back to view and formatted the package.json.